dependencies/README.md

```diff
@@ -0,0 +1,62 @@
+Static Linkage Checker
+======================
+
+Static Linkage Checker is a tool that inspects classpaths for [static linkage check](
+../library-best-practices/glossary.md#static-linkage-check).
+The tool identifies static linkage errors that appear in Java projects at runtime.
+It scans the class files in a classpath provided as input for references
+to other classes. Then it verifies compatibility of the methods and fields
+in the referenced classes, with regard to their type signatures and accessors.
+
+### Use Cases
+
+There are three use cases for Static Linkage Checker:
+
+-  **For organizations** that provide multiple libraries developed by different teams,
+  the tool helps to ensure that there are no static linkage conflicts among the libraries and their
+  dependencies when some of them are used at the same time.
```

Comment:
Remove "when some of them are used at the same time." - this is implied by "among the libraries and their dependencies".